src/julia-syntax.scm

```diff
@@ -3999,7 +4020,7 @@ f(x) = yt(x)
                                    cvs)))
                `(new_opaque_closure
                  ,(cadr e) (call (core apply_type) (core Union)) (core Any)
-                 (opaque_closure_method (null) ,nargs ,isva ,functionloc ,(convert-lambda lam2 (car (lam:args lam2)) #f '() (symbol-to-idx-map cvs)))
+                 (opaque_closure_method (null) ,nargs ,isva ,functionloc ,(convert-lambda lam2 (car (lam:args lam2)) #f '() (symbol-to-idx-map cvs) parsed-method-stack))
                  ,@var-exprs))))
           ((method)
            (let* ((name  (method-expr-name e))
```

Comment:
Is this the only place that we encounter a `'method`?

If so, I think you might be able to add to the stack here directly and get rid of the extra `(cl-convert- ...)` wrapper

Reply:
I prefer the wrapped versions because they make it really clear that "this function is supposed to allocate the parsed-method-stack", "this function is supposed to handle the push/pop stack discipline" and "this function is supposed to actually convert the closure".

I'd be more inclined to keep the wrappers, though that's a bit subjective.

Reply:
The push/pop discipline should be automatic now (that's one advantage of the native flisp form)

The `set!` in the wrapper doesn't actually need un-setting since it only affects the local binding (iiuc)

Reply:
Sounds good. Removed the `set!` but kept the wrapper.